Front-End/public/styles.css

```diff
@@ -1,29 +1,204 @@
 body {
-    background-color: grey;
-    color: black;
-    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial;
+    margin: 0;
+    width: 100%;
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+
+
+/*navigation bar*/
+.navbar {
+    background-color: #ffffff;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5% 5% 0.5% 5%;
```

Comment:
Using percentages for padding in this case isn't the best idea as it's not clear what this means(a percentage of what and why?). Either use pixels or something else.